### ISSUE TITLE
Webpack incremental build does not work

### ISSUE THREAD

**Issue by Leekao** [danez/grunt-webpack#246]
According to Webpack5 documentation in order to utilize incremental build I need to pass `cache` object with `type: "filesystem"` however that does nothing when I pass it through Grunt-webpack and the documentation for Grunt-webpack refer to `cache` as a Boolean.

**Comment by Leekao** [comment]
If anyone in the future wonders how I resolved this issue the solution for me was to remove `grunt-webpack` and to run webpack from cli using `grunt-shell` and it indeed builds incrementally.

**Comment by evenstensberg** [comment]
Makes sense. Grunt suppress output and input a lot more than needed. Like error handling.
